node/src/components/small_network.rs

```diff
@@ -237,15 +237,19 @@ where
         }
 
         // If given consensus key configuration, load it for handshake signing.
-        let consensus_keys = consensus_cfg
-            .map(|cfg| {
-                let root = cfg.dir();
-                cfg.value()
-                    .load_keys(root)
-                    .map(|(secret_key, public_key)| ConsensusKeyPair::new(Arc::try_unwrap(secret_key).expect("did not expect additional references on freshly created secret key Arc"), public_key))
-            })
-            .transpose()
-            .map_err(Error::LoadConsensusKeys)?;
+        let consensus_keys =
+            consensus_cfg
+                .map(|cfg| {
+                    let root = cfg.dir();
+                    cfg.value().load_keys(root)
+                })
+                .transpose()
+                .map_err(Error::LoadConsensusKeys)?
+                .map(|(secret_key, public_key)| {
+                    ConsensusKeyPair::new(Arc::try_unwrap(secret_key).expect(
+                    "did not expect additional references on freshly created secret key Arc",
+                ), public_key)
```

Comment:
```suggestion
                        "did not expect additional references on freshly created secret key Arc",
                    ), public_key)
```
(…or is this how rustfmt did it?)

Reply:
I'll double check :)

Reply:
It seems `rustfmt` has been waving the white flag facing this one. I manually improved the formatting.